### PR TITLE
Color edit without tooltip fix

### PR DESCRIPTION
**Pull request by mstabrin**
# Description

The QColorSwatch class had a typo in the assignment of the tooltip if no explicit tooltip was provided.

I fixed the typo by removing the second underscore

```
self.setToolTip(tooltip or trans.__('click to set color'))
to
self.setToolTip(tooltip or trans._('click to set color'))
```

## Type of change

- [X] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References
None

# How has this been tested?

- [X] I wrote two tests creating the respective classes with default values and explicitly provided values 
- [X] all tests pass with my change

## Final checklist:
- [X] My PR is the minimum possible work for the desired functionality
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (Czaki) [inline napari/_qt/widgets/_tests/test_qt_color_swatch.py]: it will be nice if you use `pytest.mark.parametrize` instead of inner for loop.

you should  also add all created widgets to cleaning using `qtbot.add_widget(widget)`

**Review by Czaki — CHANGES_REQUESTED** [review]

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (Czaki) [inline napari/_qt/widgets/_tests/test_qt_color_swatch.py]: same as above

**Comment by mstabrin** [comment]
@Czaki I actually really did not like the inner for loop. Thank you for pointing me in the right direction :)

 I changed it to use `pytest.mark.parametrize` and added all created widgets to the `qtbot`.

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (mstabrin) [reply napari/_qt/widgets/_tests/test_qt_color_swatch.py]: Done

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (mstabrin) [reply napari/_qt/widgets/_tests/test_qt_color_swatch.py]: Done

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (Czaki) [inline napari/_qt/widgets/_tests/test_qt_color_swatch.py]:
```suggestion
    widget = QColorSwatchEdit(parent, initial_color=color, tooltip=tooltip)
    qtbot.add_widget(widget)
    assert widget.tooltip() == tooltip or 'click to set color'
    assert widget.color() == color or TRANSPARENT 
```

With the import of transparent from the proper file.

File: napari/_qt/widgets/_tests/test_qt_color_swatch.py
Comment (Czaki) [inline napari/_qt/widgets/_tests/test_qt_color_swatch.py]: similar change?

**Review by Czaki — COMMENTED** [review]
As `parent` is only passed to the super class constructor I suggest removing this parameter as we should our code, not Qt itself. 

As it looks simple we may also add tests to verify if object is properly initialized? Pleas see what you think about my suggestion

**Comment by mstabrin** [comment]
@Czaki I think that makes a lot of sense.

I modified your suggestion to check explicitly for the expected value depending on the input for `tooltip` and `color`.

**Review by Czaki — APPROVED** [review]
> I modified your suggestion to check explicitly

did you know about using the `or` operator during the evaluation? 

```python
a = b or c
```
if `b` evaluate to `True` then assign `b` to `a` else assign `c` to `a` 

Such code is shorter and function (for me will be more readable). But I lest this decision for you and maybe the napari maintainer. The current state is good for me.

**Comment by mstabrin** [comment]
@Czaki Thanks for the hint, I did not use the `or` statement this way, yet. I will go with your suggestion :)

**Comment by mstabrin** [comment]
@Czaki I am still not sure how merging works :) do I need to write someone and ask for the merge and that you reviewed it or do I just need to wait?

**Review by Carreau — APPROVED** [review]
Looks good to me, thanks for the pull request and the reviews.

**Comment by Carreau** [comment]
> I am still not sure how merging works :) do I need to write someone and ask for the merge and that you reviewed it or do I just need to wait?

Usually someone with merge right will check PRs every now and then, and merge if all looks good. Feel free to ping or ask for more-reviews/feedback if you have not got any for a week or so. You can either comment on the issue or ask on the napari zulip.

**Comment by Czaki** [comment]
> @Czaki I am still not sure how merging works :) do I need to write someone and ask for the merge and that you reviewed it or do I just need to wait?

there is a rule in napari that it PR has only one approval the person who gives approval should wait at least 24h before merging to give possibility for other core-devs to review changes. 

But GitHub has no option: "remind me after 24h" so it happens that PR is missed by reviewing person. So feel free to ping reviewing person if at least 24 hours have passed and such person does not call another person for review.

**Pull request merged**
